airflow/contrib/hooks/salesforce_hook.py

```diff
@@ -73,19 +73,21 @@ def get_conn(self):
             )
         return self.conn
 
-    def make_query(self, query):
+    def make_query(self, query, include_deleted=False, **kwargs):
```

Comment:
I think that it would be better to use something like `query_params: dict` instead of `**kwargs` to indicate that a user can pass additional parameters too query method.